vector/src/main/java/im/vector/app/core/di/MavericksViewModelModule.kt

```diff
@@ -447,6 +448,11 @@ interface MavericksViewModelModule {
     @MavericksViewModelKey(AccountCreatedViewModel::class)
     fun accountCreatedViewModelFactory(factory: AccountCreatedViewModel.Factory): MavericksAssistedViewModelFactory<*, *>
 
+    @Binds
+    @IntoMap
+    @MavericksViewModelKey(OnboardingViewModel::class)
+    fun ftueViewModelFactory(factory: OnboardingViewModel.Factory): MavericksAssistedViewModelFactory<*, *>
```

Comment:
Maybe rename the fun to `onBoardingViewModelFactory`

Reply:
will update in the next PR to avoid having to wait for the CI to finish twice :smile:

Reply:
9ccad11642ec00b67fcf47ac383bc4f9b5705ff1